hashes/src/sha3_256/benches.rs

```diff
@@ -0,0 +1,33 @@
+use test::Bencher;
+
+use crate::{Hash, HashEngine};
+
+#[bench]
+pub fn sha3_256_10(bh: &mut Bencher) {
+    let mut engine = crate::sha3_256::Hash::engine();
+    let bytes = [1u8; 10];
+    bh.iter(|| {
+        engine.input(&bytes);
+    });
+    bh.bytes = bytes.len() as u64;
```

Comment:
I ack'ed because this code mimics code already in the codebase but FTR I have no clue what this does. The field is public on `Bencher` but has no docs and from the source of a single file I can't see either. Does anyone know off the top of their head, I'm too lazy to look further into the source code for `test`. Feel free to ignore me.

Reply:
IIRC it controls the "XYZ MB/sec" output of `bench`.

In general, we need to move away from this crufty benchmark framework to criterion or something. As of [Rust 1.88.0](https://releases.rs/docs/1.88.0/) they are taking `#[bench]` away from us (well, they "destabilized" it, which maybe doesn't affect us since our benchmarks have been nightly-only for years..).